<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QLineEdit" name="lineEdit_mask">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Введите маску файлов</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_del">
      <property name="text">
       <string>удалить по завершению</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_input">
      <property name="text">
       <string>Входные данные</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEdit_input">
        <property name="text">
         <string>D:\atest\input_test</string>
        </property>
        <property name="placeholderText">
         <string>Введите директорию</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_input">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLineEdit" name="lineEdit_bit">
      <property name="text">
       <string>11111111</string>
      </property>
      <property name="placeholderText">
       <string>Введите битовую маску</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_out">
      <property name="text">
       <string>Выходные данные</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="pushButton_start">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Начать</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_file_name">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>имя</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_progress">
          <property name="text">
           <string>Прогресс</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="12" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_bit">
      <property name="text">
       <string>Битовая маска</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit_out">
        <property name="text">
         <string>D:\atest\output_test</string>
        </property>
        <property name="placeholderText">
         <string>Введите директорию</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_out">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_mask">
      <property name="text">
       <string>Маска файлов</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>lineEdit_bit</zorder>
   <zorder>pushButton_start</zorder>
   <zorder>pushButton_out</zorder>
   <zorder>label_mask</zorder>
   <zorder>lineEdit_out</zorder>
   <zorder>lineEdit_mask</zorder>
   <zorder>lineEdit_input</zorder>
   <zorder>label_input</zorder>
   <zorder>label_out</zorder>
   <zorder>checkBox_del</zorder>
   <zorder>label_bit</zorder>
   <zorder>pushButton_input</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>line</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
